docs/src/index.md

```diff
@@ -180,9 +180,18 @@ linear (generic function with 2 methods)
 
 julia> grads = gradient(() -> sum(linear(x)), Params([W, b]))
 Grads(...)
+```
+To inspect the `Grads(...)` object returned for implicit parameters, you can access it using the parameters passed to `Params`:
 
+```julia
+# Apply gradients to model parameters
 julia> grads[W], grads[b]
```

Comment:
I can't make a suggestion block, but I think this "To inspect" should be removed, to restore one continuous code block which can be copy-pasted. The description of what grads is is covered below.

I also think "Apply gradients" sounds wrong, sounds like this is performing gradient descent or something. These are simply extracting elements of this container. I'm not sure it needs more description than what's said just below it, but if you think it does, can we find better words?

Reply:
Are you okay with my suggestion block above? (just to keep this less confusing for @JordiBolibar)

Reply:
Yes I think so!

Sorry this is probably a maximally confusing review, we all seem to be typing in parallel...

I mark this resolved.